src/Timer.tsx

```diff
@@ -45,24 +36,35 @@ class Timer extends React.Component<TimerProps, TimerState> {
   }
 
   tick = async () => {
+    console.log(this.props, this.state);
```

Comment:
Assuming WIP, but probably dont want to print the state every tick lol

Reply:
Good catch